docs/sections/user_guide/cli/Makefile

```diff
@@ -1,4 +1,4 @@
-SUBDIRS = $(shell find . -maxdepth 1 -mindepth 1 -type d)
+SUBDIRS = $(shell find . -maxdepth 1 -mindepth 1 -type d | sort)
```

Comment:
Doesn't matter much, but a deterministic processing order can be helpful when debugging.